release-notes/6.0/known-issues.md

```diff
@@ -1,4 +1,4 @@
-# .NET 6.0 RC2 Known Issues
+# .NET 6.0 Known Issues
```

Comment:
```suggestion
# .NET 6 Known Issues
```

Reply:
Should I do the same across these files then?
https://github.com/dotnet/core/blob/main/release-notes/5.0/5.0-known-issues.md